packages/styles/components/_c.toggle.scss

```diff
@@ -83,6 +84,7 @@
           "notification-available-16",
           $color-toggle-on-background
         ));
+        background-size: contain;
```

Comment:
~~background-size: contain;~~ => Due to the "cascade", it is not necessary to repeat this value here.